Makefile

```diff
@@ -15,8 +15,16 @@ GOLANG_VERSION?="1.16"
 GO ?= $(shell source ./scripts/common.sh && build::common::get_go_path $(GOLANG_VERSION))/go
 GO_TEST ?= $(GO) test
 
+ifdef CODEBUILD_SOURCE_VERSION ## If in codebuild environment, use the branch-specific bundle manifest URL for non-main (e.g. release branch) sources
```

Comment:
to remove the extra if block? `CODEBUILD_SOURCE_VERSION?=main`

Reply:
If you do this, a simple way to make this work locally would be just to check it in as release-0.6 in that release branch?

Reply:
lol too easy